evm-types.md

```diff
@@ -591,7 +598,7 @@ The local memory of execution is a byte-array (instead of a word-array).
     syntax ByteArray ::= ByteArray "[" Int ".." Int "]" [function]
  // --------------------------------------------------------------
     rule WS [ START .. WIDTH ] => substrBytes(padRightBytes(WS, START +Int WIDTH, 0), START, START +Int WIDTH) requires START <Int #sizeByteArray(WS) [concrete]
-    rule WS [ START .. WIDTH ] => padRightBytes(.Bytes, WIDTH, 0)                                              [owise, concrete]
+    rule WS [ START .. WIDTH ] => padRightBytes(.Bytes, WIDTH, 0)                                              [owise]
```

Comment:
I'm curious: Why did you remove `concrete` here, but not for the rule above?

Reply:
Removing `concrete` allows it to simplify in more cases to the empty bytearray, otherwise there were specific lemmas added which simplified in cases that it could. This is just more general than all those lemmas were.